internal/client/client.go

```diff
@@ -105,6 +105,10 @@ func (a *AuthApiClient) Ping() ybmclient.ApiGetPingRequest {
 	return a.ApiClient.HealthCheckApi.GetPing(a.ctx)
 }
 
+func (a *AuthApiClient) GetUser() ybmclient.ApiGetUserRequest {
+	return a.ApiClient.UserApi.GetUser(a.ctx)
```

Comment:
Trying to understand: Where do you want to add GetUser() and also this seems irrelevant to this PR.